recipes/_configure.rb

```diff
@@ -140,6 +140,14 @@
   end
 end
 
+bash "remove_old_authorizer" do
+  user "root"
```

Comment:
nitpick use `cwd node['kkafka']['libs_dir']` instead of calling `cd` in the script

Reply:
fixed